lib/build.api.js

```diff
@@ -110,13 +110,30 @@ class Build {
         this.aquifer.console.log(this.options.symlink ? 'Creating symlinks...' : 'Copying files and directories...');
         let links = [];
         let project = this.aquifer.project;
+        const destPaths = {
+          '7': {
+            'profiles': 'profiles/',
+            'drush': 'sites/all/drush',
+            'modules': 'sites/all/modules',
+            'themes': 'sites/all/themes',
+            'site': 'sites/default'
+          },
+          '8': {
+            'profiles': 'profiles',
+            'drush': 'drush',
+            'modules': 'modules',
+            'themes': 'themes',
+            'site': 'sites/default'
+          }
+        };
+        const versionDestPaths = destPaths[project.drupalVersion];
```

Comment:
I think this needs to be a class property. That way, when extensions can easily support d7/8 without having to create their own drupal version specific path maps.